src/osd/modules/debugger/win/consolewininfo.cpp

```diff
@@ -90,11 +265,19 @@ consolewin_info::consolewin_info(debugger_windows_interface &debugger) :
 }
 
 
+//-------------------------------------------------
+//  dtor
+//-------------------------------------------------
+
 consolewin_info::~consolewin_info()
 {
 }
 
 
+//-------------------------------------------------
+//  set_cpu
```

Comment:
This is another comment that adds no value, just clutter, additional work to update, and risk of getting out of sync with the actual code.